wandb/sdk/launch/sweeps/utils.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import wandb
+from wandb import util
+from wandb.sdk.launch.utils import LaunchError
+
+
+def load_launch_sweep_cli_params(
+    launch_config: Optional[str], queue: Union[str, None]
+) -> Tuple[Dict[str, Any], str]:
+    if launch_config:
+        parsed_config = util.load_json_yaml_dict(launch_config)
+        if parsed_config is None:
+            raise LaunchError(f"Invalid format for launch config at {launch_config}")
```

Comment:
it would be quite user friendly if you could provide more information about the problem wih the config (if it is reasonable to do)

Reply:
oh i see it is a general couldn't load the config error... the message seems confusing... because i read it as some of the fields the match the expected launch config